test/integration/main.test.js

```diff
@@ -0,0 +1,109 @@
+process.env.NODE_ENV = 'test';
+
+const chai = require('chai');
+
+const should = chai.should();
+const request = require('supertest');
+
+
+const app = require('../../app/main');
+const knex = require('../../db');
+
+describe('routes : orders', () => {
+  beforeEach((done) => {
+    knex.migrate.rollback()
```

Comment:
if you return promise, then you don't need pass done param and call it.
done is helpful with callbacks